discovery/kubernetes/pod_test.go

```diff
@@ -312,6 +319,7 @@ func TestPodDiscoveryUpdate(t *testing.T) {
 					"__meta_kubernetes_pod_ip":        "1.2.3.4",
 					"__meta_kubernetes_pod_host_ip":   "2.3.4.5",
 					"__meta_kubernetes_pod_ready":     "unknown",
+					"__meta_kubernetes_pod_uid":       "",
```

Comment:
Out of curiosity, is a `Pod` ever created without `UID`?

Reply:
Just above this line, a Pod with an empty UID is created.  I've fixed it.